### PR TITLE
WIP Lighthouse data per theme

### PR DESCRIPTION
**Pull request by dgrammatiko**

### CODE REVIEW

File: scripts/capture-lighthouse.js [hunk 1]
```diff
@@ -58,38 +74,36 @@ const lh = async (data) => {
...
+          fs.writeFileSync(dataFile, JSON.stringify(lightHouseData));
         }
       }
 
     })
-    .catch(function(error) {
+    .catch(function (error) {
       // handle error
-      console.log(error);
+      console.log(d);
```
Comment (C-Lodder) [inline scripts/capture-lighthouse.js]: `console.log(error)`

Comment (dgrammatiko) [reply scripts/capture-lighthouse.js]: `d` will log eg
```js
{"url":"https://demo.joomlashine.com/joomla-templates/jsn_ares_2/pro/","replace":true,"save":false}
```
`error` will log something cryptic from axios. I think is better to know which url failed rather that there was a timeout or whatever...
We could log both...

Comment (C-Lodder) [reply scripts/capture-lighthouse.js]: Ahh sorry, I didn't see the definition of `d`. Nice variable name btw ;)

Comment (dgrammatiko) [reply scripts/capture-lighthouse.js]: `d` is the code for `inputData`. Remove the `input` and scrap the `ata`... 😂

**Pull request merged**